lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -1689,6 +1689,16 @@ def test_set_zlim():
         ax.set_zlim(top=0, zmax=1)
 
 
+@image_comparison(["shared_view.png"], style="mpl20")
```

Comment:
This seems like a test that could do a check figures equal instead of a full image comparison.

Reply:
Yup, that works well, took out the baseline image.